client/blocks/plugins-scheduled-updates-multisite/schedule-list-cards.tsx

```diff
@@ -29,6 +31,15 @@ export const ScheduleListCards = ( props: Props ) => {
 					key={ schedule.id }
 				/>
 			) ) }
+			{ schedules.length === 0 && (
+				<div className="empty-state empty-state__center">
+					<p>
+						{ translate(
+							"Oops! We couldn't find any scheduled updates based on your search criteria. You might want to check your input and try again"
```

Comment:
```suggestion
							'Oops! We couldn&#8217;t find any schedules based on your search criteria. You might want to check your search terms and try again.'
```

For future reference, let's always use `&#8217;` in translation strings to make sure we use a single curly quote ([see WordPress Core](https://github.com/search?q=repo%3AWordPress%2FWordPress+%26%238217%3B&type=code)), and end any sentences with a period.

Since they're searching schedules, I suggest changing the string to reflect that. Personal preference, "search terms" feels just a tad more specific and possibly easier/more accurate to translate.

Reply:
> For future reference, let's always use `&#8217;`

I'll remember it.

Thanks for the feedback.